File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if not nums:
            return 0

        num = set(nums)
        longest = 0

        for i in num:
            if (i - 1) not in num:
                currentnum = i
                currentstreak = 1

                while (currentnum + 1) in num:
                    currentnum += 1
                    currentstreak += 1

                longest = max(longest, currentstreak)

        return longest

File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int ans = nums.length;
        int []n=new int [2*ans];
        for(int i=0;i<ans;i++){
            n[i]=nums[i];
            n[i+ans]=nums[i];
        }
          return n;
    }
  
}